dev-cpp/gflags/gflags-2.2.2.recipe

```diff
@@ -70,6 +70,10 @@ INSTALL()
 
 	sed -i 's,\/include,/'${relativeIncludeDir}',g' \
 		$libDir/cmake/gflags/gflags-config.cmake
+	sed -i 's,\/include,/'${relativeIncludeDir}',g' \
```

Comment:
Nitpick: the replacement is the same for all 3 files, so they could all be given to a single call of sed.